cli/query/query.go

```diff
@@ -56,10 +56,11 @@ func NewCommands() []cli.Command {
 				Flags:  options.RPC,
 			},
 			{
-				Name:   "tx",
-				Usage:  "Query transaction status",
-				Action: queryTx,
-				Flags:  queryTxFlags,
+				Name:      "tx",
+				Usage:     "Query transaction status",
+				UsageText: "neo-go query tx <hash> -r endpoint [-v]",
```

Comment:
Could you re-check the commit message? Both messages seem identical to me.

Reply:
I made them the same by accident, fixed!